### ISSUE TITLE
fcntl64 symbol not found

### ISSUE THREAD

**Issue by Trinovantes** [TryGhost/node-sqlite3#1576]
On v5.0.3, I'm getting this error when installed inside `node:14-alpine` docker base image

```
Error: Error relocating /app/node_modules/sqlite3/lib/binding/napi-v6-linux-x64/node_sqlite3.node: fcntl64: symbol not found
    at Object.Module._extensions..node (internal/modules/cjs/loader.js:1144:18)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object.<anonymous> (/app/node_modules/sqlite3/lib/sqlite3-binding.js:4:15)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
```

```Dockerfile
FROM node:14-alpine
ENV NODE_ENV 'production'
WORKDIR /app

# Install dependencies
COPY yarn.lock package.json     ./
RUN yarn install
CMD yarn start
```

It seems alpine is missing some lib but shouldn't the prebuilt binaries be batteries included without requiring dynamic linking?

**Comment by bkimminich** [comment]
Same issue [here](https://github.com/juice-shop/juice-shop/runs/6011313367?check_suite_focus=true#step:3:565) before pinning `sqlite3` version to `5.0.2` instead of using `^5.0.0`. In my case a `node:16-alpine` image is affected which was [built on `node:16` and then copied over all build artifacts](https://github.com/juice-shop/juice-shop/blob/master/Dockerfile) to save space. Worked for many years and wasn't changed in any way, started breaking exactly when `sqlite3` version `5.0.3` came out.

**Comment by daniellockyer** [comment]
It seems we're missing the Linux MUSL variant of prebuilt binaries and our template string isn't specific enough to differentiate MUSL and Glibc platforms. Not sure why this wasn't a problem in v5.0.2 though

Would you be able to fallback to build, or stay on v5.0.2 for now? 🙂

**Comment by daniellockyer** [comment]
I've added MUSL builds to CI, and they'll be shipped in the next release 🙂

**Comment by bkimminich** [comment]
I've changed the `sqlite3` version from pinned `5.0.2` (which always worked and still works fine during Docker build - see https://github.com/juice-shop/juice-shop/runs/6268194166?check_suite_focus=true) to `~5.0.6` since the issue is supposedly fixed since `5.0.4`. Unfortunately then my Docker build crashes again, but for no obvious reason: https://github.com/juice-shop/juice-shop/runs/6263305356?check_suite_focus=true

Can't imagine this is a random coincidence, because pinning back to `5.0.2` made it work just fine again. Just to let you know that "something isn't right", but I don't know what it might actually be.
